apollo-router/src/plugins/connectors/handle_responses.rs

```diff
@@ -96,15 +96,16 @@ impl RawResponse {
                 parts,
                 debug_request,
             } => {
-                let inputs = key.inputs().merge(
-                    &connector.response_variables,
-                    &connector.response_headers,
-                    connector.config.as_ref(),
-                    context,
-                    Some(parts.status.as_u16()),
-                    supergraph_request,
-                    Some(&parts),
-                );
+                let inputs = key
+                    .inputs()
+                    .clone()
+                    .merger(&connector.response_variables)
+                    .config(connector.config.as_ref())
+                    .context(context)
+                    .status(Some(parts.status.as_u16()))
```

Comment:
any reason why `status()` couldn't take a `u16` instead of an `Option<u16>`? it's either called with a value, or not called

Reply:
You are right, 4c48d58ed